packages/near-membrane-dom/src/custom-devtools-formatter.ts

```diff
@@ -237,16 +275,13 @@ if (LOCKER_UNMINIFIED_FLAG) {
                 ];
             } else {
                 let currentKeyStyle = keyEnumerableStringStyleObject;
-                if (
-                    typeof ownKey === 'symbol' ||
-                    !ReflectApply(ArrayProtoIncludes, keys, [ownKey])
-                ) {
+                if (ownKey === 'length') {
```

Comment:
Why is length hardcoded here? Is it just for arrays?

Reply:
Yeap! Only for arrays. I guess I could do a comparison between Object.keys and Reflect.ownKeys, but then Symbols are automatically non-enumerable.

Reply:
A `ArrayIsArray` check on the value should be enough or if it's not array-specific

Reply:
Added the isArray check in addition to it.